menu_handler.rb

```diff
@@ -123,4 +124,8 @@ def create_rental
   def show_rental
     @app.show_rental
   end
+
+  def save_data
+    @app.save_data
+  end
```

Comment:
- [ ] An error occurs when the app attempts to save the data upon exit. The main reason is that `save_data` method does not exist on the `App` class 👍 
![image](https://github.com/ibugithub/School-Library/assets/46839548/c2831758-fed8-4c96-a7cd-03dd1dd868cb)